src/atc/cache/CachedLoader.py

```diff
@@ -0,0 +1,215 @@
+import sys
+from random import randint
+from typing import List, Optional
+
+import pyspark.sql.functions as f
+from pyspark.sql import DataFrame
+
+from atc.etl import Loader
+from atc.spark import Spark
+
+from .CachedLoaderParameters import CachedLoaderParameters
+
+
+class CachedLoader(Loader):
+    """
+    Remember to override self.write_operation and/or self.delete_operation
+    Any variable rows need to be added in this function.
```

Comment:
Make a short description of the function here.

Reply:
implemented